distributed/scheduler.py

```diff
@@ -2799,13 +2800,16 @@ def worker_objective(self, ts: TaskState, ws: WorkerState) -> tuple:
         """
         dts: TaskState
         comm_bytes: int = 0
+        xfers: int = 0
         for dts in ts.dependencies:
             if ws not in dts.who_has:
                 nbytes = dts.get_nbytes()
-                comm_bytes += nbytes
+                # amortize transfer cost over all waiters
+                comm_bytes += nbytes / len(dts.waiters)
+                xfers += 1
 
-        stack_time: float = ws.occupancy / ws.nthreads
-        start_time: float = stack_time + comm_bytes / self.bandwidth
+        stack_time: double = ws.occupancy / ws.nthreads
+        start_time: double = stack_time + comm_bytes / self.bandwidth + xfers * 0.01
```

Comment:
```suggestion
        stack_time = ws.occupancy / ws.nthreads
        start_time = stack_time + comm_bytes / self.bandwidth + xfers * 0.01
```
Remove cythonization